### PR TITLE
added table of contents

### PR DESCRIPTION
**Pull request by MohanSha**

### CODE REVIEW

**Review by jasonblais — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -10,6 +10,20 @@ It's written in Golang and React and runs as a single Linux binary with MySQL or
...
+  * [Try out Mattermost](#try-out-mattermost)
+  * [Deploy on Heroku](#deploy-on-heroku)
+  * [Deploy on a Cloud Platform via Bitnami](#deploy-on-a-cloud-platform-via-bitnami)
+  * [Install on Your Own Machine](#install-on-your-own-machine)
+  * [Native Mobile and Desktop Apps](#native-mobile-and-desktop-apps)
+  * [Get Security Bulletins](#get-security-bulletins)
+  * [Get Involved](#get-involved)
+  * [Learn More](#learn-more)
+  * [Get the Latest News](#get-the-latest-news)
+
```
Comment (justinegeffen) [inline README.md:25]: nit: Remove the extra line break here. :)

**Comment by jasonblais** [comment]
@metanerd do you know why the ee/circleci build fails for this PR, given there are no translations to check?

**Comment by jasonblais** [comment]
@MohanSha thank you for the contribution :)

**Pull request merged**